.nvmrc

```diff
@@ -0,0 +1 @@
+15
```

Comment:
LTS version is `16`. I think we should only use `14` (legacy) and `16` (for everything new). This project already uses lockfile version 2, so it should use node 16 (-> npm 8)

Reply:
Cool, I wasn't sure which one to use so went with the same as cdk-utils. I'll make sure we use 16 going forward, where lockfile version is 2 👍